lib/vanagon/component.rb

```diff
@@ -284,7 +284,11 @@ def deprecated_rewrite_url
     def get_source(workdir) # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
       opts = options.merge({ workdir: workdir })
       if url || !mirrors.empty?
-        fetch_mirrors(opts) || fetch_url(opts)
+        if ENV['VANAGON_USE_MIRRORS'] == 'n'
```

Comment:
I'd probably make this n, N, no, No, 0, false, False.

Reply:
Doesn't look like Vanagon has any existing logic for doing this sort of conversion from multiple values to a "true/false".

Is it really worth adding the additional complexity and conversion logic vs. clearly documenting "y" and "n" as the effective values? If someone sets it to "True" when the documented value is "n", is there really a reasonable expectation for that working?